src/models/sqlOutputContentProvider.ts

```diff
@@ -482,9 +482,17 @@ export class SqlOutputContentProvider {
                                 uri,
                             );
                         vscode.commands.executeCommand("queryResult.focus");
+                        const tabState =
```

Comment:
NIT: Maybe disable/hide the results tab when there are no results.

Reply:
It's already hidden when no result is available, I just missed correctly setting the tab state here.
<img width="719" alt="image" src="https://github.com/user-attachments/assets/5f8104b6-af7d-4edb-a447-92cde744892d">